### PR TITLE
API-389: Only view logged user jobs in process tracker

### PR DESCRIPTION
**Pull request by momoss**
**Description (for Contributor and Core Developer)**
[API-389](https://akeneo.atlassian.net/browse/API-389) : As Mary, I want to only see my launched jobs in the process tracker
[API-397](https://akeneo.atlassian.net/browse/API-397): As Mary, I want to only see my launched jobs in the dashboard
[API-398](https://akeneo.atlassian.net/browse/API-398): As Mary, I want to only see my launched exports/imports

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | Ok
| Review and 2 GTM                  | Ok
| Micro Demo to the PO (Story only) | Ok
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: features/job-tracker/display_jobs_execution_in_job_tracker.feature [hunk 1]
```diff
@@ -74,5 +74,4 @@ Feature: Display jobs execution in job tracker
     And I logout
     And I am logged in as "Julia"
     And I am on the job tracker page
-    When I click on the "CSV footwear product export" row
-    Then I should not see the text "Execution details - CSV footwear product export [csv_footwear_product_export]"
+    And the grid should contain 0 element
```
Comment (pierallard) [inline features/job-tracker/display_jobs_execution_in_job_tracker.feature]: Don't understand why you did that

Comment (momoss) [reply features/job-tracker/display_jobs_execution_in_job_tracker.feature]: I can remove it, the grid should be empty

File: src/Pim/Bundle/EnrichBundle/Doctrine/ORM/Repository/JobTrackerRepository.php [hunk 2]
```diff
@@ -44,6 +53,8 @@ public function createDatagridQueryBuilder()
             ->innerJoin('e.jobInstance', 'j')
             ->leftJoin('e.stepExecutions', 's')
             ->leftJoin('s.warnings', 'w')
+            ->where('e.user = :user')
+            ->setParameter('user', $this->tokenStorage->getToken()->getUser()->getUsername())
```
Comment (nidup) [inline src/Pim/Bundle/EnrichBundle/Doctrine/ORM/Repository/JobTrackerRepository.php]: you can avoid this using the grid listeners as we do for other grids to inject parameters, for instance here https://github.com/akeneo/pim-community-dev/blob/master/src/Pim/Bundle/EnrichBundle/Doctrine/ORM/Repository/AttributeRepository.php#L92

File: src/Pim/Bundle/DataGridBundle/EventListener/AddUsernameToJobTrackerGridListener.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+use Oro\Bundle\DataGridBundle\Event\BuildAfter;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+
+/**
+ * @author    Philippe Mossière <philippe.mossiere@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+class AddUsernameToJobTrackerGridListener
```
Comment (ahocquard) [inline src/Pim/Bundle/DataGridBundle/EventListener/AddUsernameToJobTrackerGridListener.php]: spec

**Comment by momoss** [comment]
[Green CI](http://core-contrib-ci.akeneo.com/blue/organizations/jenkins/akeneo%2Fpim-community-dev/detail/PR-6897/5/pipeline/93)

**Pull request merged**